plugins/Eq/EqCurve.cpp

```diff
@@ -527,10 +522,8 @@ void EqHandle::setlp48()
 
 double EqHandle::calculateGain(const double freq, const double a1, const double a2, const double b0, const double b1, const double b2 )
 {
-	const int SR = Engine::audioEngine()->outputSampleRate();
-
-	const double w = 2 * LD_PI * freq / SR ;
-	const double PHI = pow( sin( w / 2 ), 2 ) * 4;
+	const double w = sin(numbers::pi * freq / Engine::audioEngine()->outputSampleRate());
```

Comment:
You sure about this?

Reply:
There seems to be a change in mathematical calculation. That's what I meant.

Reply:
Nvm did some pen paper testing and the math checks out.

Reply:
Also there's one thing you can do, make `outputSampleRate` into a member variable with initialisation in the constructor to avoid calling the function every time. It gets a bit lengthy.

Reply:
I'd prefer to avoid making changes beyond the immediate scope of lines that already have to be updated to accommodate `lmms::numbers::*`, since that's what this PR is focused on. Is that okay?

Reply:
Ok then.